src/pymoc/plotting/interp_channel.py

```diff
@@ -31,32 +31,34 @@ def __init__(
     self.bs = self.make_func(bs, 'bs', self.y)
     self.bn = self.make_func(bn, 'bn', self.z)
 
-  def make_func(self, myst, name, xin): # Seems unecessary to define a method that already exists identically as a function, no?
+  def make_func(
+      self, myst, name, xin
+  ):    # Seems unecessary to define a method that already exists identically as a function, no?
```

Comment:
> @mbueti: It looks to me like the coupler update in module_wrapper.py will not update the surface buoyancy profile in the SO, even if a predictive module for the SO mixed-layer is used (l. 118-121) (since do_psi_bz should always be false for a Psi). Am I missing something?
> Somewhat relatedly, it seems to me that wrapper.do_psi_bz effectively is just a flag to distinguish SO vs. thermal wind modules. If so, perhaps we can use something more descriptive?

Yes, that was the intention of do_psi_bz, but that's actually not the best description of this flag, or determinant of the functionality. Let me give thought to a better naming convention and way to dynamically determine how and what we want to update.

Reply:
No, it doesn't seem to be necessary. We can do a refactor and cleanup of the plotting tools at some point soon. The changes in this file are only from the auto formatter.